src/css/blocks/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  height: 36px;
```

Comment:
What happens if the header contains many lines?

🫠

<img width="213" alt="image" src="https://user-images.githubusercontent.com/86183007/216448673-095541af-e07d-4d6f-af9f-22eca52c7152.png">

Reply:
fixed